pennylane_lightning_gpu/src/simulator/StateVectorCudaManaged.hpp

```diff
@@ -1397,7 +1397,7 @@ class StateVectorCudaManaged
 
     /**
      * @brief Apply a given host or device-stored array representing the gate
-     * `matrix` to the statevector at qubit indices given by `tgts` and
+     * `matrix` to the state vector at qubit indices given by `tgts` and
```

Comment:
Is this consistent with PennyLane and other Lightning packages use of the term?

Reply:
Good question. From what I can see, it's about half-half in the PLL-GPU source, but it was 2 vs 1 for `state vector` in `StateVectorCudaManaged.hpp`. In the PennyLane source code (e.g. docstrings) `state vector` seems favored. I could `sed` this one way or another if we want to pick a side for good :)

Reply:
Sounds good. Let's opt for `state vector` then when discussing, as keep `statevector` for class or method specific.